x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -69,10 +70,17 @@ export const HomePage: React.FC<HomePageProps> = ({ children }) => {
   const { browserFields, indexPattern, indicesExist } = useWithSource('default', indexToAdd);
 
   return (
-    <WrappedByAutoSizer data-test-subj="wrapped-by-auto-sizer" ref={measureRef}>
-      <HeaderGlobal />
-
-      <Main data-test-subj="pageContainer">
+    <EuiFlexGroup
+      direction="column"
+      gutterSize="none"
+      data-test-subj="wrapped-by-auto-sizer"
+      style={{ minHeight: 'calc(100vh - 48px)' }}
```

Comment:
Never mind that it's inline style, it was result of a lot of experimenting, will fix it when we agree on this solution.